pootle/apps/pootle_store/models.py

```diff
@@ -1720,6 +1745,8 @@ def update(self, overwrite=False, store=None, only_newer=False):
             the current DB store's FS store will be used as a source.
         :param only_newer: Whether to update only the files that changed on
             disk after the last sync.
+        :param user: User to attribute updates to.
+        :param submission_type: Submission type of saved updates.
```

Comment:
yep, my bad - but to go back now and put it into the correct commit is not trivial - apologies